sql-statements/sql-statement-create-table.md

```diff
@@ -238,8 +238,9 @@ mysql> DESC t1;
 
 ## MySQL compatibility
 
-* All of the data types except spatial types are supported.
-* `FULLTEXT`, `HASH` and `SPATIAL` indexes are not supported.
+* All of the data types except `SPATIAL` types are supported.
```

Comment:
Spatial type means https://dev.mysql.com/doc/refman/8.4/en/spatial-type-overview.html i.e. `GEOMETRY`, `POLYGON`, `MULTIPOINT` etc. There isn't a single type called `SPATIAL`, and this item is not talking about spatial index either.